packages/commonwealth/client/scripts/views/modals/AuthModal/EVMWalletsSubModal/EVMWalletsSubModal.scss

```diff
@@ -0,0 +1,61 @@
+@import '../../../../../styles/shared';
+
+.EVMWalletsSubModal {
+  background-color: black;
+
+  .CWModal {
+    border-radius: 24px;
+    box-shadow: none;
+    background: transparent;
+  }
+
+  .container {
+    background-color: #191a1a;
```

Comment:
color?

Reply:
discussing this [here](https://github.com/hicommonwealth/commonwealth/pull/6577#discussion_r1479793209)